src/KernelFunctions.jl

```diff
@@ -46,7 +46,7 @@ export spectral_mixture_kernel, spectral_mixture_product_kernel
 
 export ColVecs, RowVecs
 
-export MOInput
+export MOInput, mo_input
```

Comment:
I'd suggest to remove `MOInput` from the exports:
```suggestion
export mo_input
```